pkgs/applications/misc/keepassx/community.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchFromGitHub, fetchpatch,
-  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales
+  cmake, libgcrypt, zlib, libmicrohttpd, libXtst, qtbase, qttools, libgpgerror, glibcLocales, libyubikey, libxi, libxtst, qtx11extras
```

Comment:
They also need to be passed to `buildInputs`. `libxi` is called `libXi` in nixpkgs.